light-client/src/config/checkpoints.rs

```diff
@@ -21,13 +21,31 @@ pub struct RawSlotResponseData {
 
 #[derive(Debug, Default, Clone, PartialEq, Eq, Serialize, Deserialize)]
 pub struct Slot {
+    #[serde(deserialize_with = "str_or_u64")]
     pub slot: u64,
     pub block_root: Option<H256>,
     pub state_root: Option<H256>,
     pub epoch: u64,
     pub time: StartEndTime,
 }
 
+fn str_or_u64<'de, D>(deserializer: D) -> Result<u64, D::Error>
```

Comment:
We don't need to implement this, `ethportal-api` already [uses](https://github.com/ethereum/trin/blob/01f9a5e4e0d28e5c2f0649eca78426a07f2d2447/ethportal-api/Cargo.toml#L38) a crate [serde_this_or_that](https://docs.rs/serde-this-or-that/latest/serde_this_or_that/index.html) where [as_u64](https://docs.rs/serde-this-or-that/latest/serde_this_or_that/fn.as_u64.html) is implemented for us.